src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,98 @@
+.stopwatch {
+  display: flex;
+  width: 400px;
+  height: 400px;
+  border: 5px solid black;
+  border-radius: 50%;
+  position: relative;
+  justify-content: center;
+
+  &__hand-seconds {
+    width: 2px;
+    height: 198px;
+    border: 1px solid black;
+    background-color: $seconds-hand-color;
+    position: absolute;
+
+    animation-name: seconds;
+    animation-delay: 1s;
+    animation-duration: 5s;
+    animation-fill-mode: forwards;
+    animation-timing-function: linear;
```

Comment:
all animations must be infinite, check [this](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-iteration-count), because now the arrows stops after one iteration